meross_iot/cloud/devices/subdevices/generic.py

```diff
@@ -40,11 +40,15 @@ def online(self):
 
     def _sync_status(self):
         payload = {'all': [{'id': self.subdevice_id}]}
-        res = self._hub.execute_command('GET', HUB_MTS100_ALL, payload)
```

Comment:
When I've coded this generic sub-device, I was not aware of all the types of subdevice that could be available. It appears that we have at least MTSXXX and MSXXX. Instead of building complex IF-ELIF statement within the generic class, I'd split the implementation into specific subclasses, leaving this method dummy ("pass"). Each subclass should override this _sync_status executing the corresponding GET command.

Reply:
I left the _sync_status method in GenericSubDevice, but created a new property "_status_token" that each sub-class must define.